### ISSUE TITLE
Update existing repos on the hub to include modelcard, config file, and widget

### ISSUE THREAD

**Issue by adrinjalali** [skops-dev/skops#110]
The existing models on the hub, for example under https://huggingface.co/scikit-learn, don't necessarily include a model card or a config file and the widget on many of them doesn't work.

They should be re-generated with the latest tools to fix the issues.

When generating cards, we should also make sure there's a contact info so that the person can be contacted easily.

cc @skops-dev/maintainers

**Comment by merveenoyan** [comment]
@adrinjalali Maybe it makes sense to have their scripts in one place and add the recent features to scripts and re-run every time we add features to update the repositories. What do you say?

**Comment by adrinjalali** [comment]
Yes, that's why I also created the #111. To include the scripts in the same repo.

**Comment by adrinjalali** [comment]
@merveenoyan you've updated a few of those repositories, are you going to update the rest or should we close this one?

**Comment by merveenoyan** [comment]
@adrinjalali I'm on it, wanted to get persistence notebook out first. Will wrap them up today.

**Comment by merveenoyan** [comment]
Done, let me know if there's anything you'd like me to change.
